### PR TITLE
RHIROS-260 - Added display_name to systems/<inventory_id>

### PR DESCRIPTION
**Pull request by r14chandra**

### CODE REVIEW

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Hello @r14chandra, thank you for creating pull-request.
This looks good codewise :+1:  :ok_hand: 

Just an observation (which isn't specific to display_name change):
Inside GET request for systems list, it uses `system_columns` to send system columns` details in the response. I would like to see if we can declare a constant variable outside of GET systems list request & use it for both GET requests (i.e systems list & system details). 
System info will be consistent across these two requests after this change. What do you think?

**Comment by r14chandra** [comment]
@kgaikwad Yeah, I think it can be done. Will try.

File: ros/api/v0/hosts.py [hunk 1]
```diff
@@ -103,16 +108,12 @@ def get(self):
...
         hosts = []
-        system_columns = [
-            'inventory_id', 'fqdn', 'display_name',
-            'instance_type', 'cloud_provider',
-            'rule_hit_details', 'state', 'number_of_recommendations']
         for row in query_results:
             try:
                 system_dict = row.System.__dict__
                 host = {skey: system_dict[skey] for skey in system_columns}
+                host['fqdn'] = row.System.fqdn
```
Comment (kgaikwad) [inline ros/api/v0/hosts.py:116]: could you please add `fqdn` also in system columns list?

File: ros/api/v0/hosts.py [hunk 2]
```diff
@@ -17,6 +17,11 @@
 SYSTEM_STATES_EXCEPT_OPTIMIZED = [
     "Oversized", "Undersized", "Idling", "Storage rightsizing"
 ]
+system_columns = [
```
Comment (kgaikwad) [inline ros/api/v0/hosts.py:20]:
```suggestion
SYSTEM_COLUMNS = [
```
Constants are written in all capital letters & underscores separating the words.

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Thanks @r14chandra for updates. 
Added two small inline comments.

File: ros/openapi/openapi.json [hunk 3]
```diff
@@ -341,6 +341,9 @@
                     "inventory_id": {
                         "type": "string"
                     },
+                    "display_name": {
```
Comment (kgaikwad) [inline ros/openapi/openapi.json:344]: Along with `display_name`, needs to add `fqdn` property as well under `SystemDetails`schema.

Comment (apuntamb) [reply ros/openapi/openapi.json:344]: @kgaikwad the output for `/systems/<inventory_id>` is:
```json
{
    "inventory_id": "07d1e5ab-8f7b-41f6-ad8e-b0fd4d1e4fbf",
    "display_name": "ip-172-31-45-168.ap-south-1.compute.internal",
    "display_performance_score": {
        "cpu_score": 1,
        "memory_score": 3,
        "io_score": 1
    },
    "rating": 0,
    "number_of_recommendations": 1,
    "state": "Idling",
    "report_date": "2021-09-14",
    "instance_type": "t2.micro",
    "cloud_provider": "aws",
    "idling_time": "99.60",
    "io_wait": "0.00"
}
```
which does not include `fqdn`. Do we still want to add that in `SystemDetails` schema?

Comment (r14chandra) [reply ros/openapi/openapi.json:344]: @apuntamb It's there, I've added fqdn

Comment (apuntamb) [reply ros/openapi/openapi.json:347]: @r14chandra @kgaikwad Gotcha! I was referring to your older commit.

**Review by apuntamb — APPROVED** [review]
Tested locally, works as expected! Ack from me.
@kgaikwad do you have any additional comments?

**Review by kgaikwad — APPROVED** [review]
LGTM :+1: :ok_hand:

**Comment by kgaikwad** [comment]
Thank you @r14chandra and @apuntamb for testing the PR.

**Pull request merged**